### ISSUE TITLE
How can I get access to gradient of a tensor?

### ISSUE THREAD

**Issue by Genie-Liu** [PaddlePaddle/Paddle#18358]
I'm tring to convert this pytorch tutorial example to paddle. I think it's good for people understand the weekness of neural network.
[Adversarial Example Generation](https://pytorch.org/tutorials/beginner/fgsm_tutorial.html)

In pytroch, we can get gradient of tensor by accessing it's attribute. I'm not sure where tensor in paddle stores it's gradient and how can I get it.

**Comment by frankwhzhang** [comment]
1. fluid.default_main_program().global_block().vars 这个取出所有的变量名 可以找到想要的梯度名称
2. fetch_list = ["fc_0.b_0@GRAD"] 把梯度名称放到fetch_List中 就可以取出来
